services/src/main/java/org/keycloak/services/clientpolicy/executor/SecureSigningAlgorithmEnforceExecutor.java

```diff
@@ -61,6 +62,48 @@ public String getProviderId() {
         return componentModel.getProviderId();
     }
 
+    @Override
```

Comment:
We might not add methods for existing client registration polices. Instead of it, we could treat the re-implement client registration policies in the client policy manner, not the existing client registration manner.

We could not modify the existing merged conditions and executors when introducing the new events (in this case, `REGISTERED`, `UPDATED`, `VIEW` and `UNREGISTER`).